sw/device/tests/rstmgr_cpu_info_test.c

```diff
@@ -79,8 +79,9 @@ volatile static bool double_fault;
 void ottf_exception_handler(void) {
   if (double_fault) {
     OT_ADDRESSABLE_LABEL(kDoubleFaultSecondAddrLower);
-    addr_val = mmio_region_read32(mmio_region_from_addr(kIllegalAddr2), 0);
+    mmio_region_write32(mmio_region_from_addr(kIllegalAddr2), 0, 0);
     OT_ADDRESSABLE_LABEL(kDoubleFaultSecondAddrUpper);
+    mmio_region_write32(mmio_region_from_addr(kIllegalAddr2), 0, 0);
```

Comment:
I couldn't work out what this second write is for. It is to make sure the address of the next instruction field of the cpu info dump is known?

Reply:
o wait i dont need that anymore, let me remove it.
This was back when I thought the next_pc would still be the next instruction when it exceptioned.